src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -12419,6 +12471,471 @@ public static void Main()
+    }
+}
+""";
+            CompileAndVerify(source, expectedOutput: "-1 a 0").VerifyDiagnostics(
+                // (17,19): warning CS9068: Parameter 2 has default value '"b"' in method group and '"a"' in the target delegate type.
+                //         Del del = ProgramExtensions.PExt.M;
+                Diagnostic(ErrorCode.WRN_OptionalParamValueMismatch, "ProgramExtensions.PExt.M").WithArguments("2", @"""b""", @"""a""").WithLocation(17, 19),
+                // (17,19): warning CS9068: Parameter 3 has default value '1' in method group and 'default(long)' in the target delegate type.
+                //         Del del = ProgramExtensions.PExt.M;
+                Diagnostic(ErrorCode.WRN_OptionalParamValueMismatch, "ProgramExtensions.PExt.M").WithArguments("3", "1", "default(long)").WithLocation(17, 19));
+        }
+
+        [Fact]
+        public void InstanceMethodGroup_DefaultParameters_TargetTypeConversion()
+        {
+            var source = """
+using System;
+
+public class Program
+{
+    public int field = 10;
+    delegate string Del(string s = "a", long l = 0L);
+    public string M1(string s = "c", long l = 2L) => $"{this.field} {s} {l}";
+
+    public static void Main()
+    {
+        Program prog = new Program();
+        Del del = prog.M1;
+        Console.WriteLine(del());
+    }
+}
+""";
+            var verifier = CompileAndVerify(source, expectedOutput: @"10 a 0");
+            verifier.VerifyDiagnostics(
+                // (12,19): warning CS9068: Parameter 1 has default value '"c"' in method group and '"a"' in the target delegate type.
+                //         Del del = prog.M1;
+                Diagnostic(ErrorCode.WRN_OptionalParamValueMismatch, "prog.M1").WithArguments("1", @"""c""", @"""a""").WithLocation(12, 19),
+                // (12,19): warning CS9068: Parameter 2 has default value '2' in method group and 'default(long)' in the target delegate type.
+                //         Del del = prog.M1;
+                Diagnostic(ErrorCode.WRN_OptionalParamValueMismatch, "prog.M1").WithArguments("2", "2", "default(long)").WithLocation(12, 19));
+        }
+
+        [Fact]
+        public void ExtensionMethodGroup_DefaultParameters_InferredType()
+        {
+            var source = """
+using System;
+
+namespace ProgramExtensions {
+    public static class PExt
+    {
+        public static string M(this Program p, string s = "b", long l = 1) => $"{p.Field} {s} {l}";
+    }
+}
+
+public class Program
+{
+    public int Field = 10;
+    public static void Report(object obj) => Console.WriteLine(obj.GetType());
+
+    public static void Main()
+    {
+        var m = ProgramExtensions.PExt.M;
+        Report(m);
+        Console.WriteLine(m(new Program() { Field = 20 }));
+    }
+}
+""";
+            CompileAndVerify(source, expectedOutput:
+@"<>f__AnonymousDelegate0
+20 b 1");
+        }
+
+        [Fact]
+        public void MethodGroupInferenceCompatBreak()
```

Comment:
>MethodGroupInferenceCompatBreak [](http://example.com/codeflow?start=20&length=31)

Please add a PROTOTYPE comment that this should be added to the breaking changes doc.